<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosaicoDockWidget</class>
 <widget class="QDockWidget" name="MosaicoDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>674</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::NoDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>196</width>
         <height>632</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="crearImagenBoton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recursos.qrc">
              <normaloff>:/iconos/images/export16.png</normaloff>:/iconos/images/export16.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eliminarBoton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="recursos.qrc">
              <normaloff>:/iconos/images/delete16.png</normaloff>:/iconos/images/delete16.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
